frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -147,6 +167,9 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({
 
 type VMDetailsItemProps = {
   title: string;
+  canEdit?: boolean;
```

Comment:
These 3 props should not be needed

Reply:
they are if we agree to enable an optional edit button near the title and not for each row. waiting for @matthewcarleton response.